features/steps/formatting_steps.rb

```diff
@@ -181,6 +185,14 @@
   run_output.should start_with(yellow("P"))
 end
 
+Then(/^I should see a measuring test icon in ASCII$/) do
+  run_output.should start_with("T")
+end
+
+Then(/^I should see a yellow measuring test icon$/) do
+  run_output.should start_with(yellow("T"))
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.